.changeset/brown-walls-play.md

```diff
@@ -2,17 +2,4 @@
 'skeleton': patch
 ---
 
-The Worker Runtime for development is now considered stable. This runtime provides an environment closer to Oxygen production.
-
-Enable it in your project by adding the `--worker` flag:
-
-```diff
-"scripts": {
-  "build": "shopify hydrogen build",
-- "dev": "shopify hydrogen dev --codegen",
-+ "dev": "shopify hydrogen dev --worker --codegen",
-- "preview": "npm run build && shopify hydrogen preview",
-+ "preview": "npm run build && shopify hydrogen preview --worker",
-  ...
-}
-```
+Use the worker runtime by default when running the `dev` or `preview` commands.
```

Comment:
The idea with this changeset was to convince the users to add the `--worker` flag. With the simplified text, it feels like this is now the "default" and they don't need to do anything (and they might not know what "worker runtime" is). However, they need to add the flag, otherwise they won't be using the new runtime until the next major release.

Reply:
Got it. Sounds like we didn't really settle on our intent with the skeleton changelog, because I don't think "convincing" uptake of new features was necessarily it.

Like, this is a record that we have now changed the template and its dev script now uses `--worker` by default, and therefore that's now the default behavior for new projects. I just don't think the Skeleton template is meant to be an ongoing record of features to adopt, simply because most people won't pay as much attention to it once they've implemented their own designs.